### ISSUE TITLE
False positive on `UnusedPsalmSuppress`

### ISSUE THREAD

**Issue by RSickenberg** [vimeo/psalm#8082]
Hello folks:

I got this error:
```bash
ERROR: UnusedPsalmSuppress - web/modules/custom/<redacted>/src/Plugin/Block/HeaderBlock.php:39:1 - This suppression is never used (see https://psalm.dev/207)
      ],
    ];
```
(The two last lines are red)

Here is the link to get the code: 
https://psalm.dev/r/70a9ffdd01

Psalm version:
`Psalm 4.22.0@fc2c6ab4d5fa5d644d8617089f012f3bb84b8703`

Regards

**Comment by AndrolGenhald** [comment]
I [filled in](https://psalm.dev/r/9f433197e7) the missing methods and tested on that version and it worked fine. Can you provide a reproduction repo that I can clone to reproduce the issue?

Also, could you try running with `--no-cache`? No clue if that's the issue here, but our cache system is unfortunately very unreliable at the moment :slightly_frowning_face:

**Comment by boesing** [comment]
I had these as well when working on my psalm plugin.
Whenever psalm internal issues are thrown from a plugin, psalm does not properly interprete these issues when using `psalm-suppress`.

Can you probably disable some plugins to see if the issue disappears?
